app/helpers/devise_helper.rb

```diff
@@ -0,0 +1,15 @@
+module DeviseHelper
+  def devise_error_messages!
```

Comment:
Typically you'll see a method's name end with a `!` to signify that the method hard-triggers a command or process to kick off.  Or perhaps when something is being done with force.  Like `submit!` or `create!` or `deduct!`.  

In this case, you'd want to remove the `!`.